### PR TITLE
Add the service-serving-cert-signer CA secret

### PR DESCRIPTION
**Pull request by mrogers950**
This adds the secret and namespace for the signing cert and key needed by service-serving-cert-signer.

### CODE REVIEW

File: pkg/asset/manifests/operators.go [hunk 1]
```diff
@@ -200,6 +201,7 @@ func (m *Manifests) generateBootKubeManifests(dependencies asset.Parents) []*ass
 		"03-openshift-web-console-namespace.yaml":          []byte(bootkube.OpenshiftWebConsoleNamespace),
 		"04-openshift-machine-config-operator.yaml":        []byte(bootkube.OpenshiftMachineConfigOperator),
 		"05-openshift-cluster-api-namespace.yaml":          []byte(bootkube.OpenshiftClusterAPINamespace),
+		"09-openshift-service-signer-namespace.yaml":       []byte(bootkube.OpenshiftServiceCertSignerNamespace),
```
Comment (mrogers950) [inline pkg/asset/manifests/operators.go]: I used '09-' to match the operator's runlevel here, but I'm not sure if that is necessary.

**Comment by abhinavdahiya** [comment]
This PR is just adding the secret required by service-serving-cert-signer in a namespace; can you change the PR title and description to match that.

**Comment by mrogers950** [comment]
@abhinavdahiya sure, updated.

**Comment by abhinavdahiya** [comment]
The https://github.com/openshift/installer/pull/437/commits/7f3099b7e36fa1170e8774ff7822c8b91b4a5c5b still have the title `Bootstrap the service-serving-cert-signer CA `

**Comment by mrogers950** [comment]
@abhinavdahiya done.

**Comment by abhinavdahiya** [comment]
/retest

Registry tls timeout error

**Pull request merged**